### ISSUE TITLE
Set up unlearning evaluation with random subset removal

### ISSUE THREAD

**Issue by chris522229197** [q8888620002/Group-Attribution-for-Diffusion-Models#10]
- For an unlearned model and a retrained model corresponding to the same removed subset, evaluate the similarity (e.g., image-level L2 distance, CNN penultimate layer L2 distance) of their generated images.
- Sample a validation set of random subsets for hyperparameter tuning (e.g., number of epochs).
- Sample a held-out set of random subsets for final evaluation.
- Consider different sampling distributions (datamodel distribution, Shapley distribution, Banzhaf uniform distribution). The datamodel distribution will likely give more differentiable results, because it removes 20%, 50%, or 75% of the training at a time. For the Shapley and Banzhaf distribution, too few training samples might be removed to make a difference between the fully trained vs. retrained model.
